filewriter_v2/README.md

```diff
@@ -0,0 +1,387 @@
+| [start_time](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-start-time-field) | NX_DATE_TIME | ISO 8601 time/date of the first data point collected in UTC. |
+
+## /entry/data
+
+Describes the plottable data and related dimension scales. Please also refer to [Multi-Channel Data](#multi-channel-data) for further details.
+
+| Name | NeXus Type | Description |
+|---|---|---|
+| @NX_class | NX_CHAR | [`"NXdata"`](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-data-group) |
+| [@axes](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axes-attribute) | NX_CHAR[4] | String array that defines the independent data fields used in the default plot for all of the dimensions of the `/entry/data/data` field, `"image_number, channel, ".", "."`. |
+| [@image_number_indices](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-indices-attribute) | NX_INT | Indicates the dependency relationship of the `image_number` field with a dimension of the data, in this case `0`.
+| [@start_time_indices](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-indices-attribute) | NX_INT | Indicates the dependency relationship of the `start_time` field with a dimension of the data, in this case `0`.
+| [@channel_indices](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-indices-attribute) | NX_INT | Indicates the dependency relationship of the `channel` field with a dimension of the data, in this case `1`.
+| [@signal](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-signal-attribute) | NX_CHAR | Declares which field is the default to be plotted, `"data"`. |
+| [image_number](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-field) | NX_NUMBER[] | Dimension scale defining the axis `image_number` of the data, typically this is `[1, ..., nP]`. |
+| [start_time](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-field) | NX_NUMBER[] | Dimension scale defining the axis `start_time` of the data. These are the relative start times for all images with absolute reference `/entry/start_time`. |
+| [channel](https://manual.nexusformat.org/classes/base_classes/NXdata.html#nxdata-axisname-field) | NX_CHAR[][^2] | Dimension scale defining the axis `channel` of the data. These are the enabled channels, for example `["threshold_1", "threshold_2", "difference"]`. |
+| [data](https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-data-data-field) | NX_NUMBER[nP,ch,i,j] | The image data.<br> **Note:** Files in V2 format always have only one `"/entry/data/data"` dataset, even in case `nimages_per_file` is greater than `0`. This is because it is a [Virtual Dataset](https://portal.hdfgroup.org/display/HDF5/Virtual+Dataset++-+VDS). |
+
+## /entry/instrument
```

Comment:
Link to https://manual.nexusformat.org/classes/applications/NXmx.html#nxmx-entry-instrument-group